pyproject.toml

```diff
@@ -9,33 +9,25 @@ authors = ["Killian Lucas <killian@openinterpreter.com>"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.10"
-openai = "^0.28.0"
-rich = "^13.4.2"
-tiktoken = "^0.4.0"
+python = ">=3.10,<3.12"
+
+appdirs = "^1.4.4"
 astor = "^0.8.1"
+chromadb = "^0.4.14"
 git-python = "^1.0.3"
-tokentrim = "^0.1.9"
-appdirs = "^1.4.4"
-six = "^1.16.0"
-python-dotenv = "^1.0.0"
-
-inquirer = "^3.1.3"
-wget = "^3.2"
 huggingface-hub = "^0.17.3"
+inquirer = "^3.1.3"
 litellm = "0.8.6"
+ooba = "^0.0.21"
+openai = "^0.28.0"
+python-dotenv = "^1.0.0"
 pyyaml = "^6.0.1"
+rich = "^13.4.2"
+six = "^1.16.0"
+tiktoken = "^0.4.0"
+tokentrim = "^0.1.9"
+wget = "^3.2"
 yaspin = "^3.0.1"
-ooba = "^0.0.21"
-chroma = "^0.2.0"
```

Comment:
This is the actual package we’re removing.

I think it might have been `poetry add`ed by accident as we don’t actually seem robust it for anything and `chromadb` - also installed - is the package for the Chroma Vector Database.

Reply:
Yes, my mistake— I must have thought that was `chromadb`.